.eslintrc.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
       "always"
     ],
     "no-multiple-empty-lines": [2, {"max": 2}],
+    "space-in-parens": [2, "never"],
+    "keyword-spacing": [2, { "before": true}],
+    "keyword-spacing": [2, { "after": true}],
```

Comment:
Please also add the [space-before-blocks](http://eslint.org/docs/rules/space-before-blocks) rule as mentioned in #180.